lndclient/macaroon_pouch.go

```diff
@@ -104,5 +107,12 @@ func newMacaroonPouch(macaroonDir string) (*macaroonPouch, error) {
 		return nil, err
 	}
 
+	m.readonlyMac, err = newSerializedMacaroon(
+		filepath.Join(macaroonDir, defaultReadonlyFilename),
+	)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Yes, for the `Versioner` client. I wanted the `Versioner` to behave like every other subserver client we have. Treating it differently than the others just because for now the returned information is static probably leads to more questions in the long run. And would need more explanatory comments everywhere.
For example, I plan to add an `Unlocked` status to the version RPC that will tell the user if the wallet is unlocked or not. This information will be very useful to have dynamically.

Reply:
Interesting, yes that status would be nice to have. I hope the versioner subserver is already up before wallet unlock?

Reply:
I haven't started with the PR yet. But yes, the idea is to load selected subservers together with the walletunlocker.